tests/embedding_net_test.py

```diff
@@ -173,6 +174,46 @@ def simulator1d(theta):
     posterior.potential(s)
 
 
+@pytest.mark.parametrize("n_points", (32, 16))
+@pytest.mark.parametrize("modes", (4, 8))
+@pytest.mark.parametrize("in_channels", (1, 2))
+@pytest.mark.parametrize("conv_channels", (8, 5))
+@pytest.mark.parametrize("num_layers", (2, 3))
+def test_spectral_conf_embedding(
```

Comment:
This is essentially the same as the CNN embedding net test, which makes a lot of sense. However as we now have a lot of new embedding nets +tests coming up, it would be great to avoid the code repetition, and refactor the embedding net api test so that it can be reused for all the embedding nets. I will make a new issue for this, as it should be a separate PR